auto_py_to_exe/web/js/i18n.js

```diff
@@ -342,6 +352,7 @@ const translationMap = {
         ar: `خيارات محددة ${applicationName}`,
         fa: `گزینه‌های ویژه ${applicationName}`,
         sr: 'Precizno podešavanje auto-py-to-exe-a',
+        sk: 'Špecifické nastavenia ${applicationName}',
```

Comment:
The variable is not being inserted correctly

```suggestion
        sk: `Špecifické nastavenia ${applicationName}`,
```